invenio_app_ils/ui/src/invenio_app_ils/pages/frontsite/components/Header/Header.js

```diff
@@ -58,21 +52,42 @@ export default class Header extends Component {
 
   render() {
     return (
-      <Menu
-        stackable
-        borderless
-        fluid
-        inverted
-        fixed="top"
-        className="header-menu"
-      >
-        <Container>
-          <Menu.Item>
-            <Link to="/">ILS</Link>
-          </Menu.Item>
-          <Menu.Item position={'right'}>{this.renderRightMenuItem()}</Menu.Item>
-        </Container>
-      </Menu>
+      <>
+        <Responsive minWidth={Responsive.onlyTablet.minWidth}>
+          <Menu
+            stackable
+            borderless
+            inverted
+            fixed="top"
+            className="header-menu"
+          >
+            <Container>
+              <Menu.Item header>
+                <Link to="/">ILS</Link>
+              </Menu.Item>
+              <Menu.Menu position="right">
+                <Menu.Item>
+                  {this.renderRightMenuItem(
+                    `Hello, ${sessionManager.user.username}`
+                  )}
+                </Menu.Item>
+              </Menu.Menu>
+            </Container>
+          </Menu>
+        </Responsive>
+        <Responsive {...Responsive.onlyMobile}>
+          <Menu borderless inverted fixed="top" className="mobile-header-menu">
+            <Container>
+              <Menu.Item header>
+                <Link to="/">ILS</Link>
+              </Menu.Item>
+              <Menu.Menu position="right">
+                <Menu.Item>{this.renderRightMenuItem('')}</Menu.Item>
```

Comment:
Instead of calling it with an empty string, you can initialise it with default 
`renderRightDropDown = (userMenuText='') => {`